apps/web/pages/_app.js

```diff
@@ -1,8 +1,29 @@
 import "../styles/globals.css";
 import AppWrapper from "../AppWrapper";
 import Head from "next/head";
+import { useEffect } from "react";
+import { useRouter } from "next/router";
+
+import * as ga from "../lib/ga";
 
 function MyApp({ Component, pageProps }) {
+  const router = useRouter();
+
+  useEffect(() => {
+    const handleRouteChange = (url) => {
```

Comment:
this is a little redundant since ga.pageview is already a function that accepts a URL

could just do 
```js
router.events.on('routeChangeComplete', ga.pageview)
```